crates/sncast/tests/e2e/script/general.rs

```diff
@@ -508,3 +508,41 @@ async fn test_state_file_rerun_failed_tx() {
         .unwrap();
     assert_tx_entry_success(invoke_tx_entry, "invoke");
 }
+
+#[tokio::test]
+async fn test_using_release_profile() {
```

Comment:
It seems that this test also passes successfully without the changes from this PR (for example on the master branch). Could you check this?

Reply:
it fails without the changes from this PR (clean master, only this test added)

```
---- e2e::script::general::test_using_release_profile stdout ----
thread 'e2e::script::general::test_using_release_profile' panicked at crates/sncast/tests/e2e/script/general.rs:540:32:

--- Expected
++++ actual:   stdout
   1      - ...
   2      - command: script run
   3      - status: success
        1 +    Compiling lib(map_script) map_script v0.1.0 (/private/var/folders/rd/q9ymr6vd6w150wm7g3f36hf00000gn/T/.tmp5uDNi6/Scarb.toml)
        2 +    Compiling starknet-contract(map_script) map_script v0.1.0 (/private/var/folders/rd/q9ymr6vd6w150wm7g3f36hf00000gn/T/.tmp5uDNi6/Scarb.toml)
        3 +     Finished release target(s) in 2 seconds

Exit status: 0
stderr:

command: script run
error: No such file or directory (os error 2)

```